precompile/contracts/rewardmanager/contract_test.go

```diff
@@ -67,29 +71,78 @@ var (
 
 				return input
 			},
-			SuppliedGas: AllowFeeRecipientsGasCost,
+			SuppliedGas: AllowFeeRecipientsGasCost + FeeRecipientsAllowedEventGasCost,
```

Comment:
q: should we check that this gas is strictly needed? I.e. that if we provide some more gas, remaining gas would be equale to the extra specified?

Reply:
We require exact gas amount supplied in tests. It requires remainingGas == 0  here https://github.com/ava-labs/subnet-evm/blob/fix-abi-get-inputs/precompile/testutils/test_precompile.go#L77